### PR TITLE
Bumped Phalcon to offical 3.2.2LTS

### PR DESCRIPTION
**Pull request by ninjapanzer**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
What's the reasoning for disallowing `--HEAD` installs?

**Comment by ninjapanzer** [comment]
@SMillerDev I may have misunderstood the purpose of `head` and `devel` options. My impression was this defined the location to pull the source for the given version. I am getting the impression my understanding is false.

**Comment by SMillerDev** [comment]
As far as I understand this is only if you specifically call for a head build. Any other option will download the latest release. (installing from source too)

**Comment by ninjapanzer** [comment]
Sounds good. I am returned the `head` options that were originally there.

**Comment by ninjapanzer** [comment]
Have I made a mistake by manually updating the SHA of the new build?

**Comment by SMillerDev** [comment]
No, that's supposed to be changed as it verifies the downloaded file

File: Formula/php71-phalcon.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Php71Phalcon < AbstractPhp71Extension
   desc "Full-stack PHP framework"
   homepage "https://phalconphp.com/"
   url "https://github.com/phalcon/cphalcon/archive/v3.2.0.tar.gz"
```
Comment (sergeyklay) [inline Formula/php71-phalcon.rb]: 3.2.2

**Comment by sergeyklay** [comment]
@ninjapanzer Could you please fix `Formula/php71-phalcon.rb`

**Comment by ninjapanzer** [comment]
@BrewTestBot test

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
Thanks @ninjapanzer

**Pull request closed without merging**